frontend/src/components/dashboard/create/create-modal/CreateModal.tsx

```diff
@@ -162,8 +164,8 @@ const CreateModal = ({
                   {/* TODO: change url to passsword protected section in guide */}
                   <OutboundLink
                     className={styles.link}
-                    eventLabel={GUIDE_URL}
-                    to={GUIDE_URL}
+                    eventLabel={i18n._(TRANSTEXT.guideUrl)}
+                    to={i18n._(TRANSTEXT.guideUrl)}
```

Comment:
We can point this to https://guide.postman.gov.sg/guide/quick-start/password-protected-email and add a new variable in `TRANSTEXT` in config.ts